src/org/jitsi/impl/neomedia/MediaStreamStatsImpl.java

```diff
@@ -1209,13 +1209,12 @@ private void setRttMs(long rttMs)
             {
                 // RemoteBitrateEstimator is a CallStatsObserver and
                 // VideoMediaStream has a RemoteBitrateEstimator.
-                MediaStream mediaStream = this.mediaStreamImpl;
+                MediaStreamImpl mediaStream = this.mediaStreamImpl;
```

Comment:
Yeah I agree with you @bbaldino. Thankfully we don't downcast anything here and the RemoteBitrateEstimator now extends TransformEngine interface.